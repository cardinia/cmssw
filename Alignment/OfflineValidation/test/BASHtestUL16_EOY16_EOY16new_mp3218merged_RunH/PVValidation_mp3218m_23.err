26-Sep-2019 12:51:02 CEST  Initiating request to open file root://eoscms.cern.ch//eos/cms/store/data/Run2016H/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/20000/EE68B31E-1B12-E911-9064-0CC47A7C3424.root
26-Sep-2019 12:51:05 CEST  Successfully opened file root://eoscms.cern.ch//eos/cms/store/data/Run2016H/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/20000/EE68B31E-1B12-E911-9064-0CC47A7C3424.root
Begin processing the 1st record. Run 282037, Event 244868133, LumiSection 123 on stream 0 at 26-Sep-2019 12:51:19.835 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 26-Sep-2019 12:56:05 CEST  Run: 282037 Event: 251949222
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 0
%MSG
Begin processing the 1001st record. Run 282037, Event 250229575, LumiSection 127 on stream 0 at 26-Sep-2019 12:56:09.156 CEST
Begin processing the 2001st record. Run 282037, Event 287320897, LumiSection 148 on stream 0 at 26-Sep-2019 13:00:17.942 CEST
Begin processing the 3001st record. Run 282037, Event 294755859, LumiSection 152 on stream 0 at 26-Sep-2019 13:04:24.807 CEST
Begin processing the 4001st record. Run 282037, Event 303014660, LumiSection 157 on stream 0 at 26-Sep-2019 13:08:40.494 CEST
Begin processing the 5001st record. Run 282037, Event 310999946, LumiSection 162 on stream 0 at 26-Sep-2019 13:12:50.655 CEST
Begin processing the 6001st record. Run 282037, Event 317465119, LumiSection 166 on stream 0 at 26-Sep-2019 13:17:11.028 CEST
Begin processing the 7001st record. Run 282037, Event 325927483, LumiSection 171 on stream 0 at 26-Sep-2019 13:21:26.863 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 26-Sep-2019 13:24:54 CEST  Run: 282037 Event: 332511131
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 8995
%MSG
Begin processing the 8001st record. Run 282037, Event 333319209, LumiSection 176 on stream 0 at 26-Sep-2019 13:25:38.451 CEST
Begin processing the 9001st record. Run 282037, Event 342896829, LumiSection 181 on stream 0 at 26-Sep-2019 13:29:49.042 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 26-Sep-2019 13:31:04 CEST  Run: 282037 Event: 345609692
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 298
%MSG
Begin processing the 10001st record. Run 282037, Event 349922648, LumiSection 186 on stream 0 at 26-Sep-2019 13:33:53.295 CEST
Begin processing the 11001st record. Run 282037, Event 359332381, LumiSection 191 on stream 0 at 26-Sep-2019 13:38:04.195 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 26-Sep-2019 13:38:47 CEST  Run: 282037 Event: 360509635
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 314
%MSG
Begin processing the 12001st record. Run 282037, Event 366504081, LumiSection 196 on stream 0 at 26-Sep-2019 13:42:13.790 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 26-Sep-2019 13:44:48 CEST  Run: 282037 Event: 371763515
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 0
%MSG
Begin processing the 13001st record. Run 282037, Event 373460749, LumiSection 201 on stream 0 at 26-Sep-2019 13:46:10.168 CEST
Begin processing the 14001st record. Run 282037, Event 382362798, LumiSection 205 on stream 0 at 26-Sep-2019 13:50:12.407 CEST
Begin processing the 15001st record. Run 282037, Event 391715171, LumiSection 210 on stream 0 at 26-Sep-2019 13:54:13.622 CEST
Begin processing the 16001st record. Run 282037, Event 397512855, LumiSection 213 on stream 0 at 26-Sep-2019 13:58:17.397 CEST
Begin processing the 17001st record. Run 282037, Event 407229229, LumiSection 218 on stream 0 at 26-Sep-2019 14:02:21.047 CEST
Begin processing the 18001st record. Run 282037, Event 413608971, LumiSection 222 on stream 0 at 26-Sep-2019 14:06:27.190 CEST
Begin processing the 19001st record. Run 282037, Event 421918365, LumiSection 226 on stream 0 at 26-Sep-2019 14:11:02.570 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 26-Sep-2019 14:14:57 CEST  Run: 282037 Event: 429192703
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 217
%MSG
Begin processing the 20001st record. Run 282037, Event 429695068, LumiSection 230 on stream 0 at 26-Sep-2019 14:15:08.346 CEST
Begin processing the 21001st record. Run 282037, Event 436761264, LumiSection 234 on stream 0 at 26-Sep-2019 14:19:15.232 CEST
Begin processing the 22001st record. Run 282037, Event 445459849, LumiSection 238 on stream 0 at 26-Sep-2019 14:23:23.619 CEST
Begin processing the 23001st record. Run 282037, Event 454401696, LumiSection 243 on stream 0 at 26-Sep-2019 14:27:40.853 CEST
Begin processing the 24001st record. Run 282037, Event 461990860, LumiSection 247 on stream 0 at 26-Sep-2019 14:31:52.905 CEST
Begin processing the 25001st record. Run 282037, Event 470740383, LumiSection 251 on stream 0 at 26-Sep-2019 14:35:52.387 CEST
Begin processing the 26001st record. Run 282037, Event 477655560, LumiSection 255 on stream 0 at 26-Sep-2019 14:39:55.354 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 26-Sep-2019 14:43:32 CEST  Run: 282037 Event: 485201500
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 0
%MSG
Begin processing the 27001st record. Run 282037, Event 486773773, LumiSection 260 on stream 0 at 26-Sep-2019 14:43:57.285 CEST
Begin processing the 28001st record. Run 282037, Event 493441228, LumiSection 263 on stream 0 at 26-Sep-2019 14:47:57.464 CEST
Begin processing the 29001st record. Run 282037, Event 501003038, LumiSection 267 on stream 0 at 26-Sep-2019 14:51:52.150 CEST
Begin processing the 30001st record. Run 282037, Event 508759222, LumiSection 272 on stream 0 at 26-Sep-2019 14:55:50.884 CEST
Begin processing the 31001st record. Run 282037, Event 516482023, LumiSection 276 on stream 0 at 26-Sep-2019 14:59:52.080 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 26-Sep-2019 14:59:55 CEST  Run: 282037 Event: 517154411
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 621
%MSG
Begin processing the 32001st record. Run 282037, Event 523249666, LumiSection 280 on stream 0 at 26-Sep-2019 15:03:52.779 CEST
Begin processing the 33001st record. Run 282037, Event 532710034, LumiSection 285 on stream 0 at 26-Sep-2019 15:07:47.809 CEST
Begin processing the 34001st record. Run 282037, Event 541121788, LumiSection 289 on stream 0 at 26-Sep-2019 15:12:08.167 CEST
26-Sep-2019 15:13:26 CEST  Closed file root://eoscms.cern.ch//eos/cms/store/data/Run2016H/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/20000/EE68B31E-1B12-E911-9064-0CC47A7C3424.root
26-Sep-2019 15:13:26 CEST  Initiating request to open file root://eoscms.cern.ch//eos/cms/store/data/Run2016H/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/20000/5E73034A-1B12-E911-A006-0025905A6068.root
26-Sep-2019 15:13:28 CEST  Successfully opened file root://eoscms.cern.ch//eos/cms/store/data/Run2016H/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/20000/5E73034A-1B12-E911-A006-0025905A6068.root
Begin processing the 35001st record. Run 282037, Event 1091959223, LumiSection 604 on stream 0 at 26-Sep-2019 15:16:10.320 CEST
Begin processing the 36001st record. Run 282037, Event 1099303468, LumiSection 609 on stream 0 at 26-Sep-2019 15:20:04.074 CEST
Begin processing the 37001st record. Run 282037, Event 1107321568, LumiSection 613 on stream 0 at 26-Sep-2019 15:24:32.941 CEST
Begin processing the 38001st record. Run 282037, Event 1115046357, LumiSection 618 on stream 0 at 26-Sep-2019 15:28:21.450 CEST
Begin processing the 39001st record. Run 282037, Event 1122695452, LumiSection 623 on stream 0 at 26-Sep-2019 15:32:02.080 CEST
Begin processing the 40001st record. Run 282037, Event 1130543774, LumiSection 627 on stream 0 at 26-Sep-2019 15:35:32.596 CEST
Begin processing the 41001st record. Run 282037, Event 1137751080, LumiSection 632 on stream 0 at 26-Sep-2019 15:38:58.271 CEST
Begin processing the 42001st record. Run 282037, Event 1146779617, LumiSection 637 on stream 0 at 26-Sep-2019 15:42:30.779 CEST
Begin processing the 43001st record. Run 282037, Event 1154703283, LumiSection 642 on stream 0 at 26-Sep-2019 15:46:02.744 CEST
Begin processing the 44001st record. Run 282037, Event 1162147656, LumiSection 646 on stream 0 at 26-Sep-2019 15:49:30.918 CEST
Begin processing the 45001st record. Run 282037, Event 1170201227, LumiSection 650 on stream 0 at 26-Sep-2019 15:53:00.418 CEST
Begin processing the 46001st record. Run 282037, Event 1176775170, LumiSection 654 on stream 0 at 26-Sep-2019 15:56:37.833 CEST
Begin processing the 47001st record. Run 282037, Event 1185012670, LumiSection 658 on stream 0 at 26-Sep-2019 16:00:19.780 CEST
Begin processing the 48001st record. Run 282037, Event 1194622607, LumiSection 663 on stream 0 at 26-Sep-2019 16:03:59.398 CEST
Begin processing the 49001st record. Run 282037, Event 1199435761, LumiSection 666 on stream 0 at 26-Sep-2019 16:08:03.761 CEST
26-Sep-2019 16:12:34 CEST  Closed file root://eoscms.cern.ch//eos/cms/store/data/Run2016H/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/20000/5E73034A-1B12-E911-A006-0025905A6068.root

=============================================

MessageLogger Summary

 type     category        sev    module        subroutine        count    total
 ---- -------------------- -- ---------------- ----------------  -----    -----
    1 OutOfBounds          -w TrackRefitter:Fi                       8        8
    2 fileAction           -s file_close                             2        2
    3 fileAction           -s file_open                              4        4

 type    category    Examples: run/evt        run/evt          run/evt
 ---- -------------------- ---------------- ---------------- ----------------
    1 OutOfBounds          282037/251949222 282037/332511131 282037/517154411
    2 fileAction           PostProcessEvent PostGlobalEndRun 
    3 fileAction           pre-events       pre-events       PostProcessEvent

Severity    # Occurrences   Total Occurrences
--------    -------------   -----------------
Warning                 8                   8
System                  6                   6

dropped waiting message count 0
[9.988MB/9.988MB][100%][==================================================][9.988MB/s]  [9.988MB/9.988MB][100%][==================================================][9.988MB/s]  
