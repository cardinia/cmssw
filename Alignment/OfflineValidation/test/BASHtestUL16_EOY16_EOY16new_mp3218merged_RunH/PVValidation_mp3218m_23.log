000 (3189599.023.000) 09/26 11:25:39 Job submitted from host: <137.138.44.75:9618?addrs=137.138.44.75-9618+[2001-1458-d00-1a--b]-9618&alias=bigbird14.cern.ch&noUDP&sock=schedd_2873_37ae_40>
...
040 (3189599.023.000) 09/26 12:50:52 Started transferring input files
	Transferring to host: <128.142.114.241:9618?addrs=128.142.114.241-9618+[2001-1458-301-90--100-500]-9618&noUDP&sock=14025_a193_12597>
...
040 (3189599.023.000) 09/26 12:50:52 Finished transferring input files
...
001 (3189599.023.000) 09/26 12:50:52 Job executing on host: <128.142.114.241:9618?addrs=128.142.114.241-9618+[2001-1458-301-90--100-500]-9618&noUDP&sock=13888_9760_3>
...
006 (3189599.023.000) 09/26 12:51:01 Image size of job updated: 236028
	119  -  MemoryUsage of job (MB)
	121800  -  ResidentSetSize of job (KB)
...
006 (3189599.023.000) 09/26 12:56:02 Image size of job updated: 571184
	552  -  MemoryUsage of job (MB)
	565212  -  ResidentSetSize of job (KB)
...
006 (3189599.023.000) 09/26 15:16:16 Image size of job updated: 575896
	559  -  MemoryUsage of job (MB)
	572144  -  ResidentSetSize of job (KB)
...
006 (3189599.023.000) 09/26 15:21:17 Image size of job updated: 577672
	561  -  MemoryUsage of job (MB)
	573916  -  ResidentSetSize of job (KB)
...
006 (3189599.023.000) 09/26 15:31:18 Image size of job updated: 579572
	563  -  MemoryUsage of job (MB)
	575800  -  ResidentSetSize of job (KB)
...
040 (3189599.023.000) 09/26 16:13:20 Started transferring output files
...
040 (3189599.023.000) 09/26 16:13:20 Finished transferring output files
...
005 (3189599.023.000) 09/26 16:13:20 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 03:18:22, Sys 0 00:00:15  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 03:18:22, Sys 0 00:00:15  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	11501  -  Run Bytes Sent By Job
	1015  -  Run Bytes Received By Job
	11501  -  Total Bytes Sent By Job
	1015  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :                 1           
	   Disk (KB)            :    10278        1   2145191 
	   Memory (MB)          :      563     2000      2000 
...
