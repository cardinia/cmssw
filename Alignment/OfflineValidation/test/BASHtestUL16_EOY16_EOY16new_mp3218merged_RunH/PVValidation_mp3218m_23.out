Job started at  Thu Sep 26 12:50:52 CEST 2019
batch dir: /pool/condor/dir_21363 release: /afs/cern.ch/cms/CAF/CMSALCA/ALCA_TRACKERALIGN/data/commonValidation/alignmentObjects/paconnor/CMSSW_10_6_0/src release base: /cvmfs/cms.cern.ch/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_0
cmsRun PVValidation_mp3218m_282037_cfg.py
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: This is DATA!
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: Applying TrackerSurfaceDeformations!
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: Running DA Algorithm!
Content of working dir is total 11M -rw-r--r--. 1 tkello zh 10M Sep 26 16:13 PVValidation_mp3218m_282037.root -rw-r--r--. 1 tkello zh 22K Sep 26 12:50 PVValidation_mp3218m_282037_cfg.py -rw-r--r--. 1 tkello zh 11K Sep 26 16:12 _condor_stderr -rw-r--r--. 1 tkello zh 522 Sep 26 12:50 _condor_stdout -rwxr-xr-x. 1 tkello zh 1015 Sep 26 12:50 condor_exec.exe -rw-------. 1 tkello zh 4.3K Sep 26 12:51 tkello.cc drwx------. 3 tkello zh 4.0K Sep 26 12:50 tmp drwx------. 3 tkello zh 4.0K Sep 26 12:50 var
Job ended at  Thu Sep 26 16:13:19 CEST 2019
